components/ui/theme-toggle.tsx

```
@@ -0,0 +1,24 @@
+import { useTheme } from "@/context/ThemeContext";
+
+export function ThemeToggle() {
+  const { theme, setTheme } = useTheme();
+
+  return (
+    <div className="relative">
+      <button
+        onClick={() => setTheme(theme === "dark" ? "light" : "dark")}
+        className="text-gray-300 hover:text-white transition-colors"
+        aria-label="Toggle theme"
+      >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)